### ISSUE TITLE
Please run Babel or similar for dist

### ISSUE THREAD

**Issue by hmedney** [saikojosh/Object-Assign-Deep#4]
First, thanks for this useful library 😃.  Minor suggestion - please consider running the source through Babel or similar before publishing such that the the distributable code is all ES5.  This will eliminate js compatibility issues when bundling and minifying this package (ran into an issue recently with backticks in this package).  Thanks!

**Comment by robomc** [comment]
Yeah those backticks wasted a couple of hours of my life.

**Comment by saikojosh** [comment]
OK I'll setup Babel when I get time. I didn't anticipate people would want to use this on frontend, my bad!

**Comment by mattotodd** [comment]
I just spent an hour of my life trying to track down an IE11 error: `SCRIPT1014: Invalid character`

Was related to the backticks in this library

**Comment by PabloDiablo** [comment]
Yep just hit this issue myself when running production code through Uglify: 
**Unexpected character '`'**

**Comment by gReis89** [comment]
@saikojosh any update on this? I saw a PR opened on July, can we have it merged and released?

**Comment by saikojosh** [comment]
I've converted the module to single quotes now so should work fine. When I have more time I will look at the Babel PR.

**Comment by thelastfantasy** [comment]
![default](https://user-images.githubusercontent.com/6077584/38770039-3fa60d44-403f-11e8-9e13-62a7b3884921.png)

Same problem, my project created with create-react-app got err, any idea to solve this problem?

**Comment by saikojosh** [comment]
Which version of this module are you using? I removed backticks from the latest version. 


Regards,
-Josh Cole.

**Comment by thelastfantasy** [comment]
0.4.0, I downloaded it with npm. I ran babel CLI to output an es5 version and it's ok now.

**Comment by shockey** [comment]
Chiming in here - integrating `object-assign-deep` broke our module (`swagger-client`) for Node 4 users due to use of `const` in the published module: https://github.com/swagger-api/swagger-js/issues/1315. It breaks in IE11 too, but our Babel sidestepped that problem.

Please consider transpiling 😄

**Comment by saikojosh** [comment]
@shockey Node 4 went end-of-life a month ago. I plan to merge in one of the PRs but I don't have time to do a full review right now. It's not a priority seeing as Node 4 is no longer supported.

![output](https://user-images.githubusercontent.com/1827451/39753632-694d1aae-52b7-11e8-98e6-a11afdda8015.png)

**Comment by mkanako** [comment]
my project is webpack4+vue create by vue-cli 3.0
I know this package have es6 syntax and babel igonre all files inside node_modules by default
so need to add `transpileDependencies: ['object-assign-deep']` to vue.config.js
but error also happens unfortunately
`Cannot assign to read only property 'exports' of object '#<Object>'`

I have no way😭

**Comment by shockey** [comment]
@mkanako my fork may solve your issue, it’s the same aside from the fact that it ships already transpiled through Babel: https://www.npmjs.com/package/@kyleshockey/object-assign-deep

**Comment by mkanako** [comment]
@shockey thank you very much!!
you solved my problem immediately!!
